source/_components/ps4.markdown

```diff
@@ -53,9 +52,40 @@ There are varying methods to perform this, dependent on your OS that is running
   If your Home Assistant device is running <b>Hass.io</b> on <b>HassOS</b>, it does not require additional configuration.
 </p>
 
-- Example for Debian:
-`sudo setcap 'cap_net_bind_service=+ep' /usr/bin/python3.5`
-Replace "/usr/bin/python3.5" with your path to Python that is running Home Assistant.
+### {% linkable_title Debian-based %}
+Home Assistant installed on a Debian-type OS may require configuration. This section is applicable but not limited to the following operating systems:
+
+- Debian
+- Hassbian
+- Rassbian
+- Armbian
+- Ubuntu
+
+In terminal run the following command:
+
+`$ sudo setcap 'cap_net_bind_service=+ep' <python>`
```

Comment:
Please use [this kind](https://help.github.com/en/articles/creating-and-highlighting-code-blocks) of format for code in markdown.

Reply:
Removed $ prefix from code

Reply:
I think you must do more then that, I will change somthing as example

Reply:
How about now?